README.md

```diff
@@ -84,7 +85,7 @@ top directory use the command:
 
 ## Contributing
 
-Contributions are most welcome. These fall into three main categories:
```

Comment:
Still only three categories (translation is one category).

Reply:
Sorry, I do not get this ;-)

Reply:
Sorry, you're quite right, ignore my comment. Actually, I think this is an anti-pattern in documentation, I would suggest "These fall into the following categories:".

Reply:
oki, changed